### PR TITLE
create separate class (MessageDialogWithScrollableMessage)

### PR DESCRIPTION
**Pull request by paulgheorghecristian**
# Conflicts:
#	com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java

### CODE REVIEW

File: com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package com.cubrid.common.ui.common.dialog;
+
+
```
Comment (ribram) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java]: remove one empty line

File: com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.FillLayout;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Shell;
+
+
```
Comment (ribram) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java]: remove one empty line?

File: com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    @Override
+    public Control createDialogArea(Composite parent) {
+    	if(this.message.length() == 0){
+    		return super.createDialogArea(parent);
+    	}
+    	Composite content = (Composite) super.createDialogArea(parent);
+    	content.setLayout(new GridLayout());
+        GridData data = new GridData(SWT.FILL, SWT.FILL, true, true);
```
Comment (ribram) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java]: is GridData necessary?

File: com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java [hunk 4]
```diff
@@ -358,12 +359,19 @@ public static GridLayout createGridLayout(int columns) {
 	 */
 	public static int openMsgBox(Shell sh, int dialogImageType, String title, String msg,
 			String[] dialogButton) {
+		return openMsgBox(sh, dialogImageType, title, msg, "", dialogButton);
+	}
+	
+	public static int openMsgBox(Shell sh, int dialogImageType, String title, String informativeMessage, String actualMessage,
```
Comment (ribram) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java]: I am not sure about the way we handle informative message. please verify that all references of openInformationBox have the same behavior.

**Review by ribram — CHANGES_REQUESTED** [review]

File: com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java (hunk 4 above)
Comment (paulgheorghecristian) [reply com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java]: What do you mean, exactly? If i'm not mistaken, it has the same behaviour as before.

File: com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java (hunk 3 above)
Comment (paulgheorghecristian) [reply com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/MessageDialogWithScrollableMessage.java]: yes. without it, the scrollbar doesn't appear ( you cannot scroll at all).

**Pull request closed without merging**